Adafruit_BluefruitLE/bluez_dbus/gatt.py

```diff
@@ -21,6 +22,8 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
+from builtins import str
+from builtins import map
 from past.builtins import map
```

Comment:
You're importing `map` twice here. Since you are wrapping all `map` statements in `list`, this line is unnecessary.

Reply:
Are both 26 and 27 unnecessary?

Reply:
Keep 26, it will make things faster https://python-future.org/compatible_idioms.html#map